docs/source/onnx.md

```diff
@@ -40,39 +38,18 @@ torch.onnx.export(
 )
 ```
 
-Next sections introduce the two versions of the exporter.
+## PyTorch2-based ONNX Exporter
 
-## TorchDynamo-based ONNX Exporter
+*The PyTorch2-based ONNX exporter is the newest exporter for PyTorch 2.6 and newer*
 
-*The TorchDynamo-based ONNX exporter is the newest (and Beta) exporter for PyTorch 2.1 and newer*
-
-TorchDynamo engine is leveraged to hook into Python's frame evaluation API and dynamically rewrite its
+PyTorch 2.0 engine is leveraged to hook into Python's frame evaluation API and dynamically rewrite its
 bytecode into an FX Graph. The resulting FX Graph is then polished before it is finally translated into an
 ONNX graph.
 
 The main advantage of this approach is that the [FX graph](https://pytorch.org/docs/stable/fx.html) is captured using
 bytecode analysis that preserves the dynamic nature of the model instead of using traditional static tracing techniques.
```

Comment:
Looks like this would not be accurate for non-strict? Maybe we can just link to torch.export docs.

Reply:
Yup that's correct.

> Maybe we can just link to torch.export docs.

What are your opinions on calling it torch.export based onnx exporter?

Reply:
That works - whatever the torch.export team recommends :)